.github/workflows/test.yaml

```diff
@@ -7,10 +7,10 @@ on:
 
 jobs:
     build:
-        name: Docs build
+        name: Docs Build
```

Comment:
why did you rename all the jobs? :D i dont think we want title case (or whatever is this style called) in here, lets not touch unrelated parts

Reply:
Sorry, OCD. It was random, some names were title cased and some not. I felt like most are title cased, so I unified it, but I have no other opinions about it and I'm fine making the effort to restrain myself to touch those parts 😅